frontend/public/components/resource-quota.jsx

```diff
@@ -453,10 +750,69 @@ export const ResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(
   },
 );
 
-export const ResourceQuotasDetailsPage = (props) => (
-  <DetailsPage
-    {...props}
-    menuActions={resourceQuotaMenuActions}
-    pages={[navFactory.details(Details), navFactory.editYaml()]}
-  />
-);
+export const AppliedClusterResourceQuotasPage = ({ namespace, mock, showTitle }) => {
+  const { t } = useTranslation();
+  const resources = [
+    {
+      kind: referenceForModel(AppliedClusterResourceQuotaModel),
+      namespaced: true,
+      namespace,
+      optional: true,
+    },
+  ];
+
+  return (
+    <MultiListPage
+      ListComponent={AppliedClusterResourceQuotasList}
+      resources={resources}
+      label={t(AppliedClusterResourceQuotaModel.labelPluralKey)}
+      namespace={namespace}
+      flatten={flatten}
+      title={t(AppliedClusterResourceQuotaModel.labelPluralKey)}
+      mock={mock}
+      showTitle={showTitle}
+    />
+  );
+};
+
+export const ResourceQuotasDetailsPage = (props) => {
+  return (
+    <DetailsPage
+      {...props}
+      menuActions={resourceQuotaMenuActions}
+      pages={[navFactory.details(Details), navFactory.editYaml()]}
+    />
+  );
+};
+
+export const AppliedClusterResourceQuotasDetailsPage = (props) => {
+  const { match } = props;
+  return (
+    <DetailsPage
+      {...props}
+      menuActions={[
```

Comment:
It would be good to avoid duplicating the actions list for the list page and details page.